src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -219,37 +214,37 @@ def generate_MCP_1_default_config(self, filename):
     }
 
     # Generate SAS Test Harnesses dump records for multiple iterations   
-    sas_test_harness_0_dump_records_iteration_0 = {
+    sas_test_harness_0_iteration_0_dump_records = {
```

Comment:
I have asked several times for you to unify the convention: (iter #, item #) vs. (item #, iter #); you keep saying that it is "done" but the problem still exists. Please pick one convention and use it for all vars in this function.

Reply:
updated   followed iter#, item# through out the configurations